hudi-common/src/main/java/org/apache/hudi/common/model/HoodieRecord.java

```diff
@@ -160,6 +161,8 @@ public String getFieldName() {
 
   protected transient Comparable<?> orderingValue;
 
+  protected Boolean isDelete;
```

Comment:
Is it worth adding this to the serialized output of the record?

Reply:
Yes, it'll saving re-compute cost for `isDelete`.